### ISSUE TITLE
Investigate replacement marquee plugin

### ISSUE THREAD

**Issue by dasgarner** [xibosignage/xibo#1398]
This one is hardware accelerated.

https://github.com/aamirafridi/jQuery.Marquee

**Comment by dasgarner** [comment]
I have had to slip this to 1.8.7 due to time pressure for the 1.8.6 release.

**Comment by dasgarner** [comment]
There are a few issues thus far
 - it does not work well with layout scaling, right scrolling tickers are cut off the end
 - it does not work with down/up
 - the speed calculation is completely different from a marquee, meaning we'd need to apply some fudge algorithm (this is not a problem, just an observation)

**Comment by dasgarner** [comment]
Some progress has been made - I've modified the plugin so that it can have a container width/height provided to it, which can then be used to correctly position the content at start/reanimate. This fixes the positional issues in 1 and 2 above.

I've also modified the plugin so that it works when running against a browser that does not support the CSS3 animation properties.

The speed calculation is still a cause for concern - CSS transforms work on the basis of transforming from position 1 to position 2 over a particular duration. This means that the plugin calculates the time it will take to scroll the entire content and that is then the duration set on the item. Our old way to do marquees didn't care so much about the total length, it simply specified a speed at which to operate.

The most critical issue is that this new plugin doesn't appear to work on Android. When a translateX transform is applied the scaling goes out of whack
